spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/tf/Session.scala

```diff
@@ -34,9 +34,11 @@ import com.intel.analytics.bigdl.models.utils.ModelBroadcast
 import TFTensorNumeric.NumericByteString
 
 import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
 import scala.reflect.ClassTag
+import scala.util.Random
 
-abstract class Session[T: ClassTag] {
+abstract class Session[T](implicit ct: ClassTag[T]) {
```

Comment:
Is it equal to T: ClassTag?

Reply:
Yes, they are the same. I want to assign this implicit parameter to a local variable to avoid serializing the whole Session.

Reply:
you can put that method to the object